### PR TITLE
PM FCLA Trading Layout

### PR DESCRIPTION
**Pull request by bneiluj**

### CODE REVIEW

File: src/components/market.js [hunk 1]
```diff
@@ -43,7 +43,34 @@ const Market = ({
   }
 
   return (
-    <article className={cn("market")}>
+    <div className="">
+      <div className="page-heading d-sm-flex justify-content-sm-between align-items-sm-center">
+        <h2 className="title mb-3 mb-sm-0">PM FCLA</h2>
+      </div>
+      <div className="jr-card p-0 border-0">
```
Comment (bneiluj) [inline src/components/market.js]: inside below code inside loop

**Comment by bneiluj** [comment]
Screenshot 2019-07-04 at 09 06 47

Icons have not been added into the commits. 

And error on the Prop: `index.js:1375 Warning: Failed prop type: The prop `oracle` is marked as required in `Market`, but its value is `undefined`.`

**Comment by leopoldjoy** [comment]
> Screenshot 2019-07-04 at 09 06 47
> 
> Icons have not been added into the commits.
> 
> And error on the Prop: `index.js:1375 Warning: Failed prop type: The prop `oracle`is marked as required in`Market`, but its value is `undefined`.`

The images have been added as you can see here: https://github.com/flyingcarpet-network/prediction-market/tree/feature/add-trading-ux/public/assets/images

However, modifications to the `config.json` file have not been transferred.

**Comment by bneiluj** [comment]
`config.json` and `config.json.bak` are generated by API's calls or smartcontract data...

**Pull request merged**